conf/battle/drops.conf

```diff
@@ -13,6 +13,14 @@ item_auto_get: no
 // How long does it take for an item to disappear from the floor after it is dropped? (in milliseconds)
 flooritem_lifetime: 60000
 
+// Bonus to loot priority for first attacker (Note 2)
+// The first person in the damage log of a monster that is alive and on the same map gains a loot priority bonus
```

Comment:
```suggestion
// The first player in the damage log of a monster, that is alive and on the same map, gains a loot priority bonus
```

Reply:
The question is if we should not rephrase this like with the other configs.
Something like: "How many percent does the first player ... get?"

Reply:
Just look at the config right above xD

Reply:
The config above is not a (Note 2) type config. If you there's no note you need to explain a bit more and we usually phrase it as a question for that, but (Note 2) already makes it clear that you set the percent so no need to ask "How much percent?"
I will stay consistent with other (Note 2) configs.